src/pages/ExamsPage/__snapshots__/index.test.jsx.snap

```diff
@@ -738,6 +747,15 @@ exports[`ExamsPage snapshots exams and attempts loaded 1`] = `
           >
             Review Dashboard
           </a>
+          <a
+            aria-selected="false"
+            class="nav-item nav-link"
+            data-rb-event-key="allowances"
+            href="#"
+            role="tab"
+          >
+            Allowances
```

Comment:
Question: Is there a reason that "Allowances" isn't set as a message (for potential future internationalization, as with AllowanceList below) here? Is this just to be consistent with others (like Review Dashboard above)? Thanks for clarifying?

Reply:
Good question! I'm not entirely sure why the other tabs aren't translated, but yes, I did this to be consistent with them.